perf/stability/intermediary/templates/intermediary-virtualservice.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: httpbin
+spec:
+  hosts:
+  - httpbin
+  http:
+  - match:
+    - sourceLabels:
```

Comment:
I believe the plan is to deprecate `sourceLabels`, it's already hidden from the docs. Maybe the intermediary can add a header and use that for matching instead?

cc @rshriram

Reply:
I think this is a good use of `sourceLabels` @rshriram ?

Reply:
yeah.. looks like it. Sigh. Lets not mark it as deprecated any more @frankbu